packages/procaptcha-pow/src/components/ProcaptchaWidget.tsx

```diff
@@ -46,30 +46,14 @@ const Procaptcha = (props: ProcaptchaProps) => {
 	// get the state update mechanism
 	const updateState = buildUpdateState(state, _updateState);
 	const manager = useRef(
-		Manager(config, state, updateState, callbacks, props.frictionlessState),
+		Manager(config, state, updateState, callbacks, frictionlessState),
 	);
 	const captchaRef = useRef<HTMLInputElement>(null);
 
 	useEffect(() => {
-		const element = captchaRef.current;
-		if (!element) return;
-
-		const form = element.closest("form");
-		if (!form) return;
-
-		const handleSubmit = () => {
-			manager.current.resetState();
-		};
-
-		form.addEventListener("submit", handleSubmit);
```

Comment:
why has this been removed?

Reply:
See point 1 here https://prosopo.slack.com/archives/C039W3E3UNB/p1728596930956019